app/models/user.rb

```diff
@@ -12,10 +12,10 @@ class User < ApplicationRecord
 
   def self.digest string
     cost = if ActiveModel::SecurePassword.min_cost
-      BCrypt::Engine::MIN_COST
-    else
-      BCrypt::Engine.cost
-    end
+             BCrypt::Engine::MIN_COST
+           else
```

Comment:
dùng toán tử3 ngôi luôn em :D

Reply:
Dạ anh ơi. này bữa em dùng toàn tử 3 ngôi thì lỗi rubocop. Nó bảo dài quá ấy nên em sửa lại như này :D